lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -277,6 +311,17 @@ window.isRunning = function() {
   return lighthouseIsRunning;
 };
 
+// Get list of installed extensions that are enabled and can be disabled.
+// Extensions are not allowed to be disabled if they are under an admin policy.
+chrome.management.getAll(installs => {
```

Comment:
also needs to not run in devtools. Can just nest in the if statement below in this case.